src/processors/knex-processor.ts

```diff
@@ -6,6 +6,20 @@ import { KnexRecord, Operation, ResourceConstructor } from "../types";
 
 import OperationProcessor from "./operation-processor";
 
+const operators = {
+  eq: '=',
+  ne: '!=',
+  lt: '<',
+  gt: '>',
+  le: '<=',
+  ge: '>=',
+  like: 'like',
+  in: 'in',
```

Comment:
@aklkv Would be nice to have a `NOT IN` operator.

Reply:
who would it look like in query params and knex?

something like:
```
whereNot((builder) => { 
   builder. whereIn([]);
} 
```

not sure if this would work tho

Reply:
I think the query parameter would be `nin` (following `ne` as non-equal), and the SQL would be `not in`.

Reply:
ok I can do that